index.html

```diff
@@ -26,27 +45,29 @@ <h2>The Past</h2>
         </section>
         
         <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
+               
+
+            <h2> Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div style="background:teal;" class="box">Box 1</div>
```

Comment:
where we can, we really want to avoid inline styling. This got the job done, but a better solution would either be to add another class with a space ie `class = "box box1` or to do a pseudo selecter https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child